api/utils/extapi.go

```diff
@@ -0,0 +1,35 @@
+package utils
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+func FetchInfo(idParam int) string {
```

Comment:
Make this function name a bit more explicit about what type of info it's fetching